bedrock/mozorg/tests/test_views.py

```diff
@@ -432,10 +432,8 @@ def test_webmaker_mentor_signup(self, mock_post, mock_subscribe):
         self.data.update(interest='education', newsletter=True)
         self.client.post(self.url_en, self.data)
 
-        assert_false(mock_subscribe.called)
-        payload = {'email': self.contact, 'custom-1788': '1'}
-        mock_post.assert_called_with('https://sendto.mozilla.org/page/s/mentor-signup',
-                                     data=payload, timeout=2)
+        mock_subscribe.assert_called_with(self.contact, 'mozilla-learning-network', source_url=ANY)
+        assert_false(mock_post.called)
```

Comment:
IMO, there's no longer a need for these `assert_false(mock_post.called)` assertions (or associated patched objects). Before this change, `email_contribute.py` could perform either a `post` or a `subscribe`, and part of these tests was to make sure the correct action was taken.

With this change, all actions are `subscribe`. The variation to test now seems to be the newsletter name/ID, which is already tested in the `mock_subscribe.assert_called_with` assertion.

Unless @jgmize or @pmac disagree, I'd say you can just remove the `assert_false(mock_post.called)` assertions and the associated patches.